include/engine/guidance/assemble_steps.hpp

```diff
@@ -40,7 +43,7 @@ std::vector<RouteStep> assembleSteps(const DataFacadeT &facade,
                                      const bool source_traversed_in_reverse,
                                      const bool target_traversed_in_reverse)
 {
-    const double constexpr ZERO_DURACTION = 0., ZERO_DISTANCE = 0., NO_BEARING = 0.;
+    const double constexpr ZERO_DURACTION = 0., ZERO_DISTANCE = 0.;
```

Comment:
sure. looked like you added a random comment from a console command, though. like `make -C` ...